extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -37,6 +37,10 @@ namespace core {
 namespace repository {
 
 void FlowFileRepository::flush() {
+  auto opendb = db_->open();
+  if (!opendb) {
+    return;
```

Comment:
Is this okay, shouldn't we log or throw?

Reply:
an unsuccessful `open` now logs an error